static/app/utils/gettingStartedDocs/node.ts

```diff
@@ -243,7 +243,7 @@ Sentry.startSpan({
 }
 // Calls to stopProfiling are optional - if you don't stop the profiler, it will keep profiling
 // your application until the process exits or stopProfiling is called.
```

Comment:
```suggestion
// Calls to stopProfileSession are optional - if you don't stop the profiler, it will keep profiling
// your application until the process exits or stopProfiling is called.
```